assets/stylesheets/common/question-answer.scss

```diff
@@ -15,6 +15,11 @@
       border-top: 1px solid var(--primary-low);
     }
   }
+
+  // disable reactions by default
+  .discourse-reactions-actions {
```

Comment:
Is there a way we can do this without coupling this plugin to another plugin that has nothing to do with this plugin?

Reply:
I tried looking for other ways (initializers, etc) but couldn't find any. This seems to be the simplest route and can be reverted (if site admin wants to enable reactions on Q&A topics) by adding a simple theme component.

Reply:
While this is the simplest solution, it is also a solution that is brittle and un-testable. If people are installing both the Q&A plugin and reactions plugin to gether, they will unfortunately have to add a theme component to hide the reactions button if they  don't want it to show. I don't think this plugin should be coupled in anyway to other plugins.

Reply:
I see. What do you suggest here to decouple both the plugins and have reactions disabled by default on Q&A topics?

Reply:
The site owner will have to add their own theme component to hide the reactions if they do not want it. From the Q&A plugin's point of view, the reactions plugin has nothing to do with it.

Reply:
Sounds good. Let's close this PR for now then.

Reply:
@techAPJ If there is a discussion about this on dev, feel free to ping me about it cause I have strong opinions about how plugins should not be coupled with each other.